### ISSUE TITLE
Is it safe to use the bot after the 0.37.0 update?

### ISSUE THREAD

**Issue by carlimFD** [Ar1i/PokemonGo-Bot#3813]

**Comment by MTK4355** [comment]
First of all, it is never "safe" to use the bot.

To answer your question as it relates to update 0.37:
It depends on whether you have used your bot account on a device with 0.37 installed.

The latest update sends a few initial requests to the server using an unknown encryption that the bot cannot. If an account begins sending these encrypted messages, then stops sending them, your account will be flagged as suspicious and will likely be banned eventually.

In other words, if you have not used the 0.37 update, nothing has changed with regard to bot safety. Continue using the bot until login requests from the old versions of the app are refused. 

JUST DO NOT LOG IN TO A BOT ACCOUNT USING 0.37 AND THEN GO BACK TO USING THE BOT.

**Comment by B4SH4** [comment]
CAUTION: It llooks like Niantic added significant security measures to the new version. If you have updated the app on your phone and loaded a bot account DO NOT USE the bot until we know more about the auth tokens being sent from the new app. Written by MTK4355 one Coder of this Bot

**Comment by TeamRedd** [comment]
yea getting soft ban a few min ago , safe to stay low for abit , use at your own risk .

**Comment by coffeexp** [comment]
Yay!! Glad i visited issue tab today after a 3 day vacation.. :+1:

**Comment by TxShack** [comment]
I think is better to add a warning to the program so everyone understand this, like the beware you still can get banned message. Cheers!

**Comment by riautama77** [comment]
does this apply to ios?

**Comment by Andrewaj** [comment]
UK appstore have Pokemon Go version 1.5.0 and it don't have pokemon buddy functionality and havent been updated for a while,  so I suppose it don't have initialisation token functionality as well.

**Comment by Blubb123** [comment]
German update is available

**Comment by Andrewaj** [comment]
Update is available now in UK app store  Version 1.7.0.
